js-api/src/chem.ts

```diff
@@ -730,4 +719,26 @@ export namespace chem {
   export function sketcher(onChangedCallback: Function, smiles: string = ''): HTMLElement {
     return api.grok_Chem_Sketcher(onChangedCallback, smiles);
   }
+
+  export function convert(s: string, sourceFormat: string, targetFormat: string) {
+    if (sourceFormat == 'mol' && targetFormat == 'smiles') {
```

Comment:
Do not use literals in such cases, please - you will eventually inevitably make a typo :) besides, it's easier to find usages this way.